src/tests.c

```diff
@@ -3690,37 +3690,77 @@ void run_wnaf(void) {
     CHECK(secp256k1_scalar_is_zero(&n));
 }
 
+void test_ecmult_accumulate(secp256k1_sha256* acc, const secp256k1_scalar* x) {
+    /* Compute x*G in 4 different ways, serialize it uncompressed, and feed it into acc. */
```

Comment:
Just confirming, `secp256k1_ecmult_multi_var` is out of scope as a 5th way for this test?

Reply:
Nope, added!